### DISCUSSION TITLE
Unable to spin spinner (video with debug)

### DISCUSSION THREAD

**Discussion by spmtn-eo** [ppy/osu#28555] [Q&A] [answered]
I am so confused
https://github.com/ppy/osu/assets/51346584/d5483620-ac17-4307-8c12-b8f7a1aa0c4a
Please help, debug is in the video
(Running on Windows 10, Nvidia Geforce GTX 670 sadly)

**Comment by BlockHead007** [comment]
Are you... holding any of two buttons while spinning? You know, just like with sliders?

**Comment by spmtn-eo** [comment]
Just figured it out. thx to BlockHead007. I give kudos to u

**Reply by spmtn-eo** [reply, marked as answer]
I just was holding the mouse button but then I holded X and it worked

**Reply by BlockHead007** [reply]
Looks like you have mouse buttons turned off, that's why it didn't work
